Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -660,7 +700,48 @@ public AndConstraint<StringAssertions> ContainEquivalentOf(string expected, stri
             Execute.Assertion
                 .ForCondition(Contains(Subject, expected, StringComparison.CurrentCultureIgnoreCase))
                 .BecauseOf(because, becauseArgs)
-                .FailWith("Expected {context:string} to contain equivalent of {0}{reason} but found {1}.", expected, Subject);
+                .FailWith("Expected {context:string} {0} to contain the equivalent of {1}{reason}.", Subject, expected);
```

Comment:
For consistency the message of `ContainEquivalentOf` should match `CollectionAssertions.ContainEquivalentOf`.
But I would leave that change out of this PR :).

Reply:
@jnyrup  If there's nothing else, can you please approve the PR?

Reply:
I just re-read my words above and see that they might read opposite of my intentions.
I was meant to be:

Drop the changes from this PR as that makes it inconsistent with `CollectionAssertions.ContainEquivalentOf`. 
The pedantic me also think it should be *an equivalent* and not *the equivalent*, as there are several different equivalents.

Reply:
@jnyrup Wait, I'm confused. This change is so that consistency _**is**_ introduced with `CollectionAssertions.ContainEquivalentOf`.

https://github.com/fluentassertions/fluentassertions/blob/c477f8bcdb6613e47792d7729bfa253785d9384d/Src/FluentAssertions/Collections/CollectionAssertions.cs#L636

The only difference is the extra word `the`, which can be removed, or alternatively replaced with `an`.

Reply:
Yeah, now I'm confused as well.

Reply:
Now I see the other changes :)
I only saw the addition of `the`, which seems to match `StringWildcardMatchingValidator.cs:62`.